src/main/java/org/jabref/logic/bibtex/comparator/EntryComparator.java

```diff
@@ -65,10 +66,10 @@ public int compare(BibEntry e1, BibEntry e2) {
         // sorted according to last name.
         if (InternalBibtexFields.getFieldProperties(sortField).contains(FieldProperty.PERSON_NAMES)) {
             if (f1 != null) {
-                f1 = AuthorList.fixAuthorForAlphabetization((String) f1).toLowerCase();
+                f1 = AuthorList.fixAuthorForAlphabetization((String) f1).toLowerCase(Locale.ROOT);
```

Comment:
In general we should use ROOT for internal strings only. And especially in reading/writing we have to be backwards compatible.
For GUI stuff we should use the English locale for english terms, e.g. the bibtex fieldnames or entry types. 
As this effects many cases. I think we should discuss this in a devcall @JabRef/developers  WDYT?